microcosm_logging/factories.py

```diff
@@ -113,20 +112,17 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
+    fields = """{
         "loggerName": "%(name)s",
         "levelName": "%(levelname)s",
-        "message": "%(message)s",
```

Comment:
I think the canonical approach would be to create a custom [Formatter](https://docs.python.org/2/library/logging.html#logging.Formatter) or similar and change the microcosm logging configuration to use it.

That would produce an explicit/obvious implementation that handled the dict and string cases appropriately. It would also let you write test code for the formatting.